packaging/hipify-clang.txt

```diff
@@ -3,13 +3,13 @@ project(hipify-clang)
 
 install(PROGRAMS @HIPIFY_BIN_INSTALL_PATH@/hipify-clang DESTINATION @CMAKE_INSTALL_BINDIR@)
 install(PROGRAMS @HIPIFY_BIN_INSTALL_PATH@/hipify-perl DESTINATION @CMAKE_INSTALL_BINDIR@)
-install(PROGRAMS @HIPIFY_BIN_INSTALL_PATH@/findcode.sh DESTINATION @CMAKE_INSTALL_LIBEXECDIR@)
-install(PROGRAMS @HIPIFY_BIN_INSTALL_PATH@/finduncodep.sh DESTINATION @CMAKE_INSTALL_LIBEXECDIR@)
+install(PROGRAMS @HIPIFY_BIN_INSTALL_PATH@/findcode.sh DESTINATION @CMAKE_INSTALL_LIBEXECDIR@/hipify)
+install(PROGRAMS @HIPIFY_BIN_INSTALL_PATH@/finduncodep.sh DESTINATION @CMAKE_INSTALL_LIBEXECDIR@/hipify)
```

Comment:
Please, do not revert those lines. The issue is due to the changed line below.

Reply:
I'd like to unblock the auto promos and psdbs in the meantime.